README.md

```diff
@@ -2,7 +2,7 @@
   <a href="https://novu.co" target="_blank">
   <picture>
     <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/2233092/213641039-220ac15f-f367-4d13-9eaf-56e79433b8c1.png">
-    <img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280" alt="Logo"/>
+    ![Novu Logo]<img src="https://user-images.githubusercontent.com/2233092/213641043-3bbb3f21-3c53-4e67-afe5-755aeb222159.png" width="280"/>
```

Comment:
Aren't we missing here also the `alt-text`?

Reply:
the ![] is the standard for alt-text in markdown. For some reason, I wasn't able to use the standard ![] for the rest of the alt-text in the README.md.

Reply:
No worries. Wanted to double check as a11y is not my strongest field.

Reply:
if you ever want or need a11y help, just give me a shout. it's my specialty.